pilot/pkg/config/kube/gateway/deploymentcontroller.go

```diff
@@ -298,6 +298,12 @@ func (d *DeploymentController) configureIstioGateway(log *istiolog.Scope, gw gat
 	}
 	log.Info("reconciling")
 
+	var ns *corev1.Namespace
+	if d.namespaces != nil {
+		ns = d.namespaces.Get(gw.Namespace, "")
+	}
+	proxyUID, proxyGID := GetProxyIDs(ns)
```

Comment:
It will not work in multi-tenant mode, because when there is no namespace informer and `ns` is `nil`, `GetProxyIDs` will return 1337, which will cause failure on pod creation, because of missing `anyuid` SCC.

Reply:
So I think that the only option we have is to remove UID/GID 1337 from the kube-gateway template if multi-tenant mode is enabled. Then UID/GID will be set automatically to the lowest value from the assigned range.